web_api/core/test_views.py

```diff
@@ -697,6 +720,183 @@ class FakeCheckoutSession:
     assert res.json()["stripePublishableApiKey"] == settings.STRIPE_PUBLISHABLE_API_KEY
 
 
+def create_pk() -> int:
+    return random.randint(1000, 100000)
```

Comment:
Is there a way to avoid this? Seems like it could lead to flakey tests

Reply:
fixed